src/quo2/components/tags/number_tag/view.cljs

```diff
@@ -0,0 +1,24 @@
+(ns quo2.components.tags.number-tag.view
+  (:require [quo2.components.icon :as icons]
+            [quo2.components.markdown.text :as text]
+            [react-native.core :as rn]
+            [quo2.components.tags.number-tag.style :as style]
+            [quo2.theme :as quo.theme]))
+
+(defn number-tag-internal
+  [{:keys [number size-key blur? theme] :as props}]
+  (let [size (get-in style/sizes [size-key :size])]
+    [rn/view (style/container props)
+     (if (and (> size 20) (< (count number) 3))
+       [text/text
+        {:size   (if (= (get-in style/sizes [size-key :size]) 32)
```

Comment:
We already have size in line 10, We can reference it in the conditional here

Reply:
Thank you for your review @ibrkhalil!

yes, done.